builder/src/lib.rs

```diff
@@ -40,7 +40,7 @@ pub const ROM_WITH_UART: FwId = FwId {
 pub const ROM_FAST_WITH_UART: FwId = FwId {
     crate_name: "caliptra-rom",
     bin_name: "caliptra-rom",
-    features: &["emu", "no-kats"],
+    features: &["emu", "fast-rom"],
```

Comment:
shall we call it val-rom, as per today discussion ?

Reply:
I agree, changed now